test/service/FileWriterImplTest.java

```diff
@@ -0,0 +1,42 @@
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.impl.FileWriterImpl;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileWriterImplTest {
+    private static final String FILE_WRITE_TEST = "test/resources/result_write.csv";
+    private static final String FILE_READ_TEST = "test/resources/result_read.csv";
+    private static final String FILE_NOT_WRITE = "wrong/file.csv";
+    private static FileWriter fileWriterTest;
+    private static StringBuilder testReport;
+
+    @BeforeClass
+    public static void setUp() {
+        fileWriterTest = new FileWriterImpl();
+        testReport = new StringBuilder();
+        testReport.append("fruit,quantity")
+                .append(System.lineSeparator())
+                .append("banana,132")
+                .append(System.lineSeparator())
+                .append("apple,90");
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void writeWrongFile_NotOk() {
```

Comment:
Do not use uppercase afrer "_". Correct method name in this case is writeWrongFile_notOk()